src/components/Navigation.js

```diff
@@ -38,9 +42,9 @@ const Navigation = () => {
             <img src={logo} className={classes.logo} alt="Logo" />
           </Typography>
 
-          <Button className={classes.button}>My Plants</Button>
-          <Button className={classes.button}>Add Plant</Button>
-          <Button className={classes.button}>My Account</Button>
+          <Link to="/myplants" className={classes.link}><Button className={classes.button}>My Plants</Button></Link>
+          <Link to="/addplant" className={classes.link}><Button className={classes.button}>Add Plant</Button></Link>
+          <Link to="#" className={classes.link}><Button className={classes.button}>My Account</Button></Link>
```

Comment:
We should actually use the `<NavLink>` component here, so we can change the look of the Link to easily show which one is currently selected.